typescript/students/RedDwarf/main.ts

```diff
@@ -0,0 +1,44 @@
+document.head.append(Object.assign(document.createElement('style'),{textContent:String.raw`
```

Comment:
What is this sh💩t

Reply:
![image](https://user-images.githubusercontent.com/68486874/192187955-c3aaddac-fa18-4939-af3f-56995a053cea.png)